prow/jobs/kyma/kyma-compass-integration.yaml

```diff
@@ -49,54 +47,58 @@ presubmits: # runs on PRs
             - name: CLOUDSDK_COMPUTE_ZONE
               value: "europe-west4-b"
             - name: INPUT_CLUSTER_NAME
-              value: "compass-integration-periodic-on-demand"
+              value: "compass-integration"
           resources:
             requests:
               memory: 200Mi
               cpu: 80m
     branches:
       - ^master$
+    run_if_changed: "^((resources\\/compass\\S+|resources\\/compass-runtime-agent\\S+|installation\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
     labels:
-      prow.kyma-project.io/slack.skipReport: "true" # this job will be ignored by Slack reporter
+      preset-build-pr: "true"
       <<: *gke_integration_job_labels_template
     extra_refs:
       - <<: *test_infra_ref
         base_ref: master
       - <<: *compass_ref
         base_ref: master
 
-periodics:
-- name: kyma-gke-compass-integration-periodic
-  cron: "0 0 * * 1-5"
-  decorate: true
-  max_concurrency: 10
-  spec:
-    containers:
-      - image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190528-8897828
-        securityContext:
-          privileged: true
-        command:
-          - "bash"
-        args:
-          - "-c"
-          - "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-compass-integration.sh"
-        env:
-          - name: CLOUDSDK_COMPUTE_ZONE
-            value: "europe-west4-b"
-          - name: INPUT_CLUSTER_NAME
-            value: "compass-integration-periodic"
-        resources:
-          requests:
-            memory: 200Mi
-            cpu: 80m
-  branches:
-    - ^master$
-  labels:
-    <<: *gke_integration_job_labels_template
-  extra_refs:
-    - <<: *kyma_ref
-      base_ref: master
-    - <<: *test_infra_ref
-      base_ref: master
-    - <<: *compass_ref
-      base_ref: master
+postsubmits:
+  kyma-project/kyma:
+  - name: post-master-kyma-gke-compass-integration
+    decorate: true
+    path_alias: github.com/kyma-project/kyma
+    skip_report: false
+    max_concurrency: 10
+    spec:
+      containers:
+        - image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190528-8897828
+          securityContext:
+            privileged: true
+          command:
+            - "bash"
+          args:
+            - "-c"
+            - "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-compass-integration.sh"
+          env:
+            - name: CLOUDSDK_COMPUTE_ZONE
+              value: "europe-west4-b"
+            - name: INPUT_CLUSTER_NAME
+              value: "compass-integration"
+          resources:
+            requests:
+              memory: 200Mi
+              cpu: 80m
+    branches:
+      - ^master$
+    labels:
+      preset-build-master: "true"
+      <<: *gke_integration_job_labels_template
+    extra_refs:
+      - <<: *kyma_ref
+        base_ref: master
+      - <<: *test_infra_ref
+        base_ref: master
+      - <<: *compass_ref
```

Comment:
Same as above, we don't need this base ref anymore.

Reply:
Done!